src/actions/popUpStateMachine.ts

```diff
@@ -0,0 +1,15 @@
+export const SHOW_POP_UP = "SHOW_POP_UP";
+export const HIDE_POP_UP = "HIDE_POP_UP";
+
+export const showPopUp = (stateMachine) => {
+    return {
+        type: SHOW_POP_UP,
+        stateMachine
```

Comment:
Information about the instances (public member). It is incomplete for the moment.